ledger/ledgercore/validatedBlock.go

```diff
@@ -45,3 +47,48 @@ func MakeValidatedBlock(blk bookkeeping.Block, delta StateDelta) ValidatedBlock
 		delta: delta,
 	}
 }
+
+// UnfinishedBlock represents a block that has been generated, but is
+// not yet ready for proposing until FinishBlock is called.
+type UnfinishedBlock struct {
+	finalAccounts map[basics.Address]AccountData // status of selected accounts at end of block
```

Comment:
It doesn't seem like these accounts are really used in this PR. Is the intent that incentives will use them after this is merged to look up info on the proposer (e.g. if paying them wouldn't violate their min balance)?

Reply:
Yes, that's the intent.  I'd prefer to merge this into the incentives PR, and perform that work before merging this to mainline.